examples/text_labels_and_annotations/unicode_minus.py

```diff
@@ -5,7 +5,7 @@
 
 By default, tick labels at negative values are rendered using a `Unicode
 minus`__ (U+2212) rather than an ASCII hyphen (U+002D).  This can be controlled
-by setting :rc:`axes.unicode_minus` (which defaults to True).
+by setting :rc:`axes.unicode_minus`.
```

Comment:
We should re-write this sentence somewhat. With the default there, it seems like it's saying you should set it to the default. But the sentence is only trying to say set it to _anything_ (manually.)